specs/integration/search/CreateReuseWidget.js

```diff
@@ -0,0 +1,18 @@
+contributeLink: 'a[title="Contribute!"]',
+publishReuseLink: ".actions-list a:first-child + a",
+nextLink: "button.pointer:last-child",
+titleField: "input#title",
+urlField: "input#url",
+descriptionField: "textarea#description",
+setTypeOption: 'select#type option[value="post"]',
+openFileUploaderLink: '.image-picker input[type="file"]',
+
+fill: function fill(name, url, description) {
```

Comment:
Actually, I think the doc is right. If you need to reuse this widget in some other feature, you'll probably want to call an action.